python/samples/concepts/agents/bedrock_agent/bedrock_agent_simple_chat.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+import uuid
+
+from semantic_kernel.agents.bedrock.bedrock_agent import BedrockAgent
+
+# This sample shows how to interact with a Bedrock agent in the simplest way.
+# This sample uses the following main component(s):
+# - a Bedrock agent
+# You will learn how to create a new Bedrock agent and talk to it.
+
+AGENT_NAME = "semantic-kernel-bedrock-agent"
+INSTRUCTION = "You are a friendly assistant. You help people find information."
+
+
+async def main():
+    bedrock_agent = await BedrockAgent.create(AGENT_NAME, instructions=INSTRUCTION)
+
+    # Use a uiud as the session id
+    new_session_id = str(uuid.uuid4())
```

Comment:
As a dev, do I really need to manage a required session_id for the invoke method? Couldn't the agent create a session id for me, if I don't want to provide one?